src/AppBundle/Controller/UserController.php

```diff
@@ -0,0 +1,354 @@
+            $this->get('request')->query->get('pageUsrs', 1), 50,/*limit per page*/
+            array(
+                'pageParameterName' => 'pageUsrs',
+                'sortFieldParameterName' => 'sortUsrs',
+                'sortDirectionParameterName' => 'directionUsrs',
+            )
+        );
+
+        $paginDshs = $this->get('knp_paginator')->paginate($dshs,
+            $this->get('request')->query->get('pageDshs', 1), 50,/*limit per page*/
+            array(
+                'pageParameterName' => 'pageDshs',
+                'sortFieldParameterName' => 'sortDshs',
+                'sortDirectionParameterName' => 'directionDshs',
+            )
+        );
+        // END PAGINATION //
+
+        // START FORM ADD USER //
+        $newUser = new User();
+
+        $form = $this->get('form.factory')->create(new UserType(), $newUser);
+
+        if ($form->handleRequest($request)->isValid()) {
+
+            // $file stores the uploaded image file
+            /** @var Symfony\Component\HttpFoundation\File\UploadedFile $file */
+            $file = $newUser->getImage();
+
+            // Generate a unique name for the file before saving it
+            $fileName = md5(uniqid()) . '.' . $file->guessExtension();
+
+            // Move the file to the directory where images are stored
+            $file->move(
+                $this->getParameter('img_directory'),
+                $fileName
+            );
+
+            // Update the 'image' property to store the img file name
+            // instead of its contents
+            $newUser->setImage($fileName);
+
+            $em = $this->getDoctrine()->getManager();
+
+            $em->persist($newUser);
+            $em->flush();
+
+            return $this->redirectToRoute('app_admin_profile', array('id' => $id));
+        }
+        // END FORM ADD USER //
+
+        // START
+        // BUILD FORM TO EDIT MY USER INFO //
+//        $form4 = $this->get('form.factory')->create(new UserType(), $usr);
+
+        // START
+        // BUILD FORM TO EDIT MY USER INFO //
+        $form2 = $this->get('form.factory')->create(new UserType(), $usr);
+
+        // START
+        // BUILD FORM TO EDIT ALL USERS INFO //
+        // RENDER VARIABLES AND PAGE //
+        return $this->render('@App/App/adminProfile.html.twig', array(
+            'id' => $id,
+            'usr' => $usr,
+            'usrs' => $usrs,
+            'dsh' => $dsh,
+            'dshs' => $dshs,
+            'paginUsrs' => $paginUsrs,
+            'paginDashs' => $paginDshs,
+            'form' => $form->createView(),
+//            'form4' => $form4->createView(),
+            'form2' => $form2->createView(),
+            'newUser' => $newUser
+        ));
+    }
+
+
+    public function deleteDashboardAction($usrid, $id)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $usr = $this->container->get('security.context')->getToken()->getUser();

```

Comment:
Now uses deleteDashAction in DashboardController. deleteDashboardAction is deleted.